### PR TITLE
Update dependencies for Node8

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -196,6 +191,9 @@
       "modules": false,
       "location": "pancake/js/",
       "name": "pancake.min.js"
+    },
+    "react": {
```
Comment (marcoalfonso) [inline package.json]: What is this for?

Comment (maxious) [inline package.json]: pancake-react!

File: src/bundles/SellerRegistration/components/SignupForm/SignupForm.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```
Comment (matt-sm) [inline src/bundles/SellerRegistration/components/SignupForm/SignupForm.js]: we should remove (and a few other files) this as its now in `app/`

Comment (maxious) [inline src/bundles/SellerRegistration/components/SignupForm/SignupForm.js]: need to remove from buyer app and buyer app tests first.

**Pull request merged**